src/public/js/startData.js

```diff
@@ -0,0 +1,79 @@
+
+export const startData = class StartData {
```

Comment:
так лучше не делать. Лучше сделать так:
```suggestion
class StartData {
.....
}

export const startData = new StartData()
```

Reply:
а у тебя в index.js еще есть const startData = ....   дубль?